app/helpers/orders_helper.rb

```diff
@@ -0,0 +1,17 @@
+# Orders Helper
+module OrdersHelper
+  def pickup_time_options
+    start_time =(Time.now + 1.hour).beginning_of_hour
+    end_time = start_time.end_of_day
```

Comment:
This can be a little hard,
but this should be according to branch closing time.

Reply:
Done